src/components/20-molecules/file-upload/demo.js

```diff
@@ -90,6 +91,55 @@ storiesOf('Examples/File Upload/Pure HTML', module)
     `;
 
     const wrapper = document.createElement('div');
+    render(template, wrapper);
+    return wrapper;
+  })
+  .add('invalid', () => {
+    const wrapper = document.createElement('div');
+
+    setTimeout(() => {
+      document.querySelector('axa-button').addEventListener('click', () => {
+        const textSelector = document.querySelector('axa-text');
+        if (document.querySelector('axa-file-upload').hasAttribute('invalid')) {
+          textSelector.innerHTML =
+            'could not submit because file-upload is invalid';
+          textSelector.style.color = '#c91432';
+        } else {
+          textSelector.innerHTML = 'submitted successfully';
+          textSelector.style.color = '#000000';
+        }
+      });
+    });
+
+    const template = html`
+      <div style="width: 455px">
+        <axa-file-upload
+          inputFileText="Upload file"
+          maxSizeOfSingleFileKB="100"
+          maxSizeOfAllFilesKB="500"
+          maxNumberOfFiles="10"
```

Comment:
I think you can delete most of the attributes because they are already set as default